src/olympia/zadmin/tests/test_views.py

```diff
@@ -77,6 +77,20 @@ def test_get_index(self):
         assert response.context['user'].email == 'admin@mozilla.com'
 
 
+class TestStaffAdmin(TestCase):
+    def test_index(self):
+        url = reverse('staffadmin:index')
+        response = self.client.get(url)
+        self.assert3xx(response, '/admin/staff-models/login/?'
+                                 'next=/en-US/admin/staff-models/')
+
+        user = user_factory(username='staffperson', email='staffperson@m.c')
+        self.grant_permission(user, 'Addons:Edit')
+        self.client.login(email='staffperson@m.c')
+        response = self.client.get(url)
+        assert response.status_code == 200
```

Comment:
you don't seem to get a 403, rather a redirect to the login page https://github.com/mozilla/addons-server/pull/5886/commits/cdfbeccb4f74f6db4b48a4b7d134d5b3b265cce6